src/components/Menu/TopCodeMenu.js

```diff
@@ -12,9 +12,9 @@ const TopCodeMenu = ({ languages }) => {
   const showCodeMenu = languages && languages.length > 1;
   const showDefaultLink = pageLanguage !== DEFAULT_LANGUAGE;
   return (
-    <div className="fixed pt-64 right-0 z-10 w-full items-end">
+    <div className="fixed right-0 z-10 mt-64 w-full items-end">
       {showCodeMenu ? (
-        <HorizontalCodeMenu>
+        <TopHorizontalMenu>
```

Comment:
I know it is not your code, but `<></>` is better than `null` and will type better when we move to TS

Reply:
Nice! I've never thought of that.